tests/page/locator-misc-2.spec.ts

```diff
@@ -150,7 +150,7 @@ it('should combine visible with other selectors', async ({ page }) => {
   await expect(page.locator('.item >> visible=true >> text=data3')).toHaveText('visible data3');
 });
 
-it('should support .visible()', async ({ page }) => {
+it('should support filter(visible)', async ({ page }) => {
```

Comment:
I realize you didn't have it before, but add a `visible: false` test?

Reply:
It is right there at the line 166.

Reply:
lol I can't read